front/src/applications/stdcmV2/components/StdcmConsist.tsx

```diff
@@ -125,8 +146,24 @@ const StdcmConsist = ({ setCurrentSimulationInputs, disabled = false }: StdcmCon
           />
         </div>
         <div className="stdcm-v2-consist__properties d-flex justify-content-between">
-          <Input id="tonnage" label={t('consist.tonnage')} trailingContent="t" />
-          <Input id="length" label={t('consist.length')} trailingContent="m" />
+          <Input
+            id="tonnage"
+            label={t('consist.tonnage')}
+            trailingContent="t"
+            type="number"
+            min={0}
+            value={totalMass === null ? undefined : totalMass}
```

Comment:
As `totalMass` has the type `number | undefined`, there is no need to check if it is equal to null.
```suggestion
            value={totalMass}
```